cfme/infrastructure/virtual_machines.py

```diff
@@ -336,9 +338,9 @@ class environment(Tab):  # noqa
             resource_pool = BootstrapSelect('environment__placement_rp_name')
             folder = BootstrapSelect('environment__placement_folder_name')
             host_filter = BootstrapSelect('environment__host_filter')
-            host_name = Table('//div[@id="prov_host_div"]/table')
```

Comment:
Q: are you sure this table was WebUI one  ?

Reply:
Doesn't the diff seem like it?

Reply:
Since we have an open PR to replace the old web_ui in this module, I'm willing to allow this for now. Otherwise new views should have new widgets. (I know, we must have missed this earlier?)